<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateWalletDialog</class>
 <widget class="QDialog" name="CreateWalletDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Wallet</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="wallet_name_line_edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Wallet Name</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="encrypt_wallet_checkbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Encrypt the wallet. The wallet will be encrypted with a passphrase of your choice.</string>
   </property>
   <property name="text">
    <string>Encrypt Wallet</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="disable_privkeys_checkbox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Disable private keys for this wallet. Wallets with private keys disabled will have no private keys and cannot have an HD seed or imported private keys. This is ideal for watch-only wallets.</string>
   </property>
   <property name="text">
    <string>Disable Private Keys</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="blank_wallet_checkbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Make a blank wallet. Blank wallets do not initially have private keys or scripts. Private keys and addresses can be imported, or an HD seed can be set, at a later time.</string>
   </property>
   <property name="text">
    <string>Make Blank Wallet</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="descriptor_checkbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use descriptors for scriptPubKey management</string>
   </property>
   <property name="text">
    <string>Descriptor Wallet</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>wallet_name_line_edit</tabstop>
  <tabstop>encrypt_wallet_checkbox</tabstop>
  <tabstop>disable_privkeys_checkbox</tabstop>
  <tabstop>blank_wallet_checkbox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateWalletDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateWalletDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
